lib/list_all.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+
+class ListAll extends StatelessWidget {
+  const ListAll({super.key, required this.listOfTasks, required this.title});
+  final Set<String> listOfTasks;
+  final String title;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(title),
+      ),
+      body: listOfTasks.isEmpty
+          ? const Center(
```

Comment:
The indentation here is a bit off, from l14-28, I think it can move <- by a tab or two.

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#indent-multi-line-argument-and-parameter-lists-by-2-characters